pyproject.toml

```diff
@@ -310,13 +313,14 @@ airbyte-lib-validate-source = "airbyte.validate:run"
 
 [tool.poe.tasks]
 test = { shell = "pytest" }
+test-fast = { shell = "pytest -m 'not slow'" }
```

Comment:
Drive-by chance: add command `poe test-fast` as CLI equivalent to "fast-tets" in CI.